### ISSUE TITLE
[Python] Implement new Concept Exercise: none

### ISSUE THREAD

**Issue by DavidGerva** [exercism/v3#1031]
This issue describes how to implement the `none` concept exercise for the Python track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student what `None` means and how `None` is used in Python.

## Learning objectives

- create a `None` object via literal declaration
- check if an object `is None`
- use a `None` object as a "`False` statement"

## Out of scope

- catching/raising Exceptions

## Concepts

- `none`

## Prerequisites

- `builtin-types`

## Resources to refer to

- [The null object](https://docs.python.org/3/library/stdtypes.html#the-null-object)

### Hints

- TBD?

### After

Nothing

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

Tests should be written using unittest.TestCase and the test file named bool_basic_test.py.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

## Edits
- edited for updated naming by @yawpitch

**Comment by mohancool001** [comment]
Hey @DavidGerva : I would like to take this as my first PR. I also wanted to do it as a part of Hacktoberfest. I just went through all the contributing docs. I will try to bring up a PR exactly like it was mentioned. Is it okay?

**Comment by DavidGerva** [comment]
> Hey @DavidGerva : I would like to take this as my first PR. I also wanted to do it as a part of Hacktoberfest. I just went through all the contributing docs. I will try to bring up a PR exactly like it was mentioned. Is it okay?

Sure! Also thank you very much. I am on vacation right now and without my pc so I will be more present the next week (if I can say I was present in the last weeks 😔). BTW again thank you and feel free to reach us for any doubt or comment! 🙏

**Comment by mohanrajanr** [comment]
Hey @DavidGerva , when you are back, check out the PR : https://github.com/exercism/v3/pull/2283

I plan to take 3 more as a part of the fest. let me know what else you want me to take :) .

**Comment by BethanyG** [comment]
PR  #2283 has been merged!  🎉   Closing.
